### PR TITLE
Add new wall

### LINKED ISSUE #824: Add a wall
Add a wall

### PR DESCRIPTION
**Pull request by pythongiant**
- [x] Read and understood [see CONTRIBUTING.md](https://github.com/fossasia/labyrinth/blob/master/CONTRIBUTING.md)
- [x] Included a preview link and screenshot showing after and before the changes.
- [x] Included a description of change below.
- [x] Tried to [squash the commits][squash].

### Changes done in this Pull Request

- View my work at pythongiant.github.io/labyrinth

- Fixes #824
### Screenshots

screenshot_11

[squash]: https://stackoverflow.com/questions/5189560/squash-my-last-x-commits-together-using-git

### CODE REVIEW

**Comment by pythongiant** [comment]
@jogendra @abishekvashok

File: js/tiles.js [hunk 1]
```diff
@@ -1648,6 +1661,8 @@ const Superhero = {
             this.ground = this.createImage("tiles/rooms/floor/superheroes/character1.svg");
         },
     }),
+
+
```
Comment (abishekvashok) [inline js/tiles.js]: Empty lines

File: js/tiles.js [hunk 2]
```diff
@@ -1896,5 +1911,5 @@ const TileMesh = {
 
     }),
 
-
+
```
Comment (abishekvashok) [inline js/tiles.js]: Blank spaces

**Comment by pythongiant** [comment]
@abishekvashok done!

**Comment by pythongiant** [comment]
Can I put this task up for review?

File: js/levels.js [hunk 3]
```diff
@@ -16,7 +16,7 @@ function createFirstLevel()
     [door.newlines, door.orchestra, door.Forbidden, door.chocolate, door.guitarCase, door.winter2],
     [door.sea, door.sofa, door.outdoor, door.chessMate, door.texture, door.chessStale],
     [door.drawnbyme, Superhero.art, PlayerStartsAt(door.black), door.green, door.banner, door.threeHeads],
-    [door.staircase, door.highLow, door.new, door.wheel, door.plain, door.top],
+    [door.quirky,door.staircase, door.highLow, door.new, door.wheel, door.plain, door.top],
```
Comment (abishekvashok) [inline js/levels.js]: Please remove a tile and add another, or else a tile can't be entered.

Comment (pythongiant) [reply js/levels.js]: alright so I remove door.both from the second last line. Is it fine now?

**Comment by pythongiant** [comment]
image
@abishekvashok @jogendra this is how it looks rn

**Comment by pythongiant** [comment]
Can we get this to merge ?

**Comment by YashKumarVerma** [comment]
> Can we get this to merge ?

Without the walls ?

**Comment by pythongiant** [comment]
> > Can we get this to merge ?
> 
> Without the walls ?

The Last One didnt too

**Comment by pythongiant** [comment]
It should work now @abishekvashok @YashKumarVerma

**Comment by pythongiant** [comment]
you can remove the tag now @YashKumarVerma

**Comment by pythongiant** [comment]
image

**Comment by pythongiant** [comment]
@YashKumarVerma merge?

**Comment by abishekvashok** [comment]
Thanks

**Comment by abishekvashok** [comment]
Task link: https://codein.withgoogle.com/dashboard/task-instances/6395398710099968/

**Comment by pythongiant** [comment]
> Task link: https://codein.withgoogle.com/dashboard/task-instances/6395398710099968/

yes

**Pull request merged**